src/main/webapp/js/instructorFeedbackEdit.js

```diff
@@ -3,6 +3,10 @@ $(document).ready(function() {
     bindUncommonSettingsEvents();
     updateUncommonSettingsInfo();
     hideUncommonPanels();
+
+    $('.dropdown-menu li').click(function() {
+        showNewQuestionFrame($(this).attr('questionType'));
```

Comment:
When you switch to use an attribute prefixed with `data`, change this to use the jQuery `data()` function